index.qmd

```diff
@@ -13,17 +13,18 @@ Many data scientists understand what goes into training a machine learning or st
 
 We expect participants to have exposure to basic modeling and machine learning practice, but NOT expert familiarity with advanced ML or MLOps topics. This workshop is for you if you:
 
-- have intermediate R or Python knowledge (this will be a “choose your own adventure” workshop where you can work through the exercises in either R or Python)
+- have intermediate R or Python knowledge (this will be a "choose your own adventure" workshop where you can work through the exercises in either R or Python)
 - can read data from CSV and other flat files, transform and reshape data, and make a wide variety of graphs
 - can fit a model to data with your modeling framework of choice
-wide variety of graphs
-
 
 ## Preparation
 
-Please join the workshop with a computer that has the following installed:
+We will provide an environment with [Posit Workbench](https://posit.co/products/enterprise/workbench/) and [Posit Connect](https://posit.co/products/enterprise/connect/) available, but you also have the option to do work locally on your laptop rather than on Posit Workbench. If this is your preference, please come to the workshop with your laptop already set up.
 
-tktk
+- Clone the [repo](https://github.com/posit-conf-2023/vetiver) so you have the files and data locally
+- To use R: Install recent versions of [R](https://cran.r-project.org/) and [RStudio](https://www.rstudio.com/download), along with the packages outlined in our [`DESCRIPTION`](https://github.com/posit-conf-2023/vetiver/blob/main/DESCRIPTION) file
+- To use Python: Install recent versions of Python and [Visual Studio Code](https://code.visualstudio.com/download), along with the packages outlined in our [`requirements.txt`](https://github.com/posit-conf-2023/vetiver/blob/main/requirements.txt)
```

Comment:
```suggestion
- To use Python: Install recent versions of Python (3.7 or greater) and [Visual Studio Code](https://code.visualstudio.com/download), along with the packages outlined in our [`requirements.txt`](https://github.com/posit-conf-2023/vetiver/blob/main/requirements.txt). 
```

Reply:
Nothing I do is VSCode specific, and I expect many people will be more familiar with Jupyter Notebook or Jupyter Lab